### PR TITLE
HTBHF-1189 Added children date of births to response.

### PR DESCRIPTION
**Pull request by dwybourn**

### CODE REVIEW

File: src/main/java/uk/gov/dhsc/htbhf/smartstub/service/BenefitsService.java [hunk 1]
```diff
@@ -61,9 +67,16 @@ private BenefitDTO getBenefits(String nino, int eligibilityStatusPosition, Strin
                 .numberOfChildrenUnderOne(childrenUnderOne)
                 .numberOfChildrenUnderFour(childrenUnderFour)
                 .householdIdentifier(householdIdentifier)
+                .children(creatChildren(childrenUnderOne, childrenUnderFour))
                 .build();
     }
 
+    private List<ChildDTO> creatChildren(Integer numberOfChildrenUnderOne, Integer numberOfChildrenUnderFour) {
```
Comment (mikeSmith08) [inline src/main/java/uk/gov/dhsc/htbhf/smartstub/service/BenefitsService.java]: Should be `createChildren`.

File: src/test/java/uk/gov/dhsc/htbhf/smartstub/controller/IntegrationTestAssertions.java [hunk 2]
```diff
@@ -17,17 +21,42 @@ public static void assertSuccessfulStatusResponse(ResponseEntity<BenefitDTO> ben
...
         AssertionsForClassTypes.assertThat(fieldError.getMessage()).isEqualTo(expectedFieldMessage);
     }
+
+    public static void assertSuccessfulNumberOfChildrenResponse(ResponseEntity<BenefitDTO> benefit, Integer childrenUnderOne, Integer childrenUnderFour) {
+        assertThat(benefit.getStatusCode()).isEqualTo(OK);
+        assertThat(benefit.getBody()).isNotNull();
+        assertChildren(benefit.getBody(), childrenUnderOne, childrenUnderFour);
+    }
+
+    public static void assertChildren(BenefitDTO benefit, Integer childrenUnderOne, Integer childrenUnderFour) {
```
Comment (mikeSmith08) [inline src/test/java/uk/gov/dhsc/htbhf/smartstub/controller/IntegrationTestAssertions.java]: Can we call this `assertCorrectNumberOfChildren` so it's a bit clearer what it does?

File: src/test/java/uk/gov/dhsc/htbhf/smartstub/service/BenefitsServiceDWPTest.java [hunk 3]
```diff
@@ -62,6 +63,7 @@ void shouldReturnTwoChildrenUnderFourForMatchingNino() {
 
         assertThat(benefit.getNumberOfChildrenUnderOne()).isEqualTo(0);
         assertThat(benefit.getNumberOfChildrenUnderFour()).isEqualTo(2);
+        assertChildren(benefit, 0, 2);
```
Comment (gethyn1) [inline src/test/java/uk/gov/dhsc/htbhf/smartstub/service/BenefitsServiceDWPTest.java]: Should we be doing these tests for HMRC as well?

File: src/test/java/uk/gov/dhsc/htbhf/smartstub/controller/IntegrationTestAssertions.java (hunk 2 above)
Comment (dwybourn) [reply src/test/java/uk/gov/dhsc/htbhf/smartstub/controller/IntegrationTestAssertions.java]: Renamed

File: src/main/java/uk/gov/dhsc/htbhf/smartstub/service/BenefitsService.java (hunk 1 above)
Comment (dwybourn) [reply src/main/java/uk/gov/dhsc/htbhf/smartstub/service/BenefitsService.java]: Renamed

File: src/test/java/uk/gov/dhsc/htbhf/smartstub/service/BenefitsServiceDWPTest.java (hunk 3 above)
Comment (dwybourn) [reply src/test/java/uk/gov/dhsc/htbhf/smartstub/service/BenefitsServiceDWPTest.java]: Added tests for hmrc as well.

**Pull request merged**